.gitignore

```diff
@@ -25,3 +25,4 @@ coverage.json
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+*.DS_Store
```

Comment:
can we remove DS_STORE?
https://twitter.com/transmissions11/status/1526941983192756228?s=20&t=EWbOYKaPZph1JPHOL6_3gw

Reply:
nooo keep
fug DS_STORE